packages/dx-react-grid-bootstrap4/src/templates/table-edit-cell.css

```diff
@@ -0,0 +1,3 @@
+.dx-rg-table-edit-cell {
+  padding: 5px!important;
```

Comment:
Can we avoid important?

Reply:
I add `!imprortant` because of the bs4 `.table td` has another padding.